tests/roles/nova_adoption/defaults/main.yaml

```diff
@@ -138,6 +138,10 @@ nova_ironic_patch: |
                 [workarounds]
                 disable_compute_service_check_for_ffu=true
           {%+ for cell in renamed_cells +%}
+          {%- set source_cell_name = cell -%}
+          {%- if cell == default_cell_name -%}
```

Comment:
when we do renaming of the default cell into cell1 during adoption (which we normally do), that change will likely break things because there would be no longer 'default' cell in the database